src/huggingface_hub/hf_api.py

```diff
@@ -663,6 +663,8 @@ class Collection(ReprMixin):
             Whether the collection is private or not.
         theme (`str`):
             Theme of the collection. E.g. `"green"`.
+        url (`str`):
+            URL of the collection on the Hub.
```

Comment:
```suggestion
            URL for the collection on the Hub.
```
Slightly prefer this, but either works.